src/main/java/com/adyen/model/checkout/CreateCheckoutSessionRequest.java

```diff
@@ -0,0 +1,1541 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2021 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+
+package com.adyen.model.checkout;
+
+import java.util.Objects;
+
+import com.adyen.model.AccountInfo;
+import com.adyen.model.Address;
+import com.adyen.model.Amount;
+import com.adyen.model.applicationinfo.ApplicationInfo;
+import com.adyen.model.Name;
+import com.adyen.model.Split;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+import static com.adyen.util.Util.toIndentedString;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * CreateCheckoutSessionRequest
+ */
+public class CreateCheckoutSessionRequest {
+    public static final String SERIALIZED_NAME_ACCOUNT_INFO = "accountInfo";
+    @SerializedName(SERIALIZED_NAME_ACCOUNT_INFO)
+    private AccountInfo accountInfo;
+
+    public static final String SERIALIZED_NAME_ADDITIONAL_AMOUNT = "additionalAmount";
+    @SerializedName(SERIALIZED_NAME_ADDITIONAL_AMOUNT)
+    private Amount additionalAmount;
+
+    public static final String SERIALIZED_NAME_ADDITIONAL_DATA = "additionalData";
+    @SerializedName(SERIALIZED_NAME_ADDITIONAL_DATA)
+    private Map<String, String> additionalData = null;
+
+    public static final String SERIALIZED_NAME_ALLOWED_PAYMENT_METHODS = "allowedPaymentMethods";
+    @SerializedName(SERIALIZED_NAME_ALLOWED_PAYMENT_METHODS)
+    private List<String> allowedPaymentMethods = null;
+
+    public static final String SERIALIZED_NAME_AMOUNT = "amount";
+    @SerializedName(SERIALIZED_NAME_AMOUNT)
+    private Amount amount;
+
+    public static final String SERIALIZED_NAME_APPLICATION_INFO = "applicationInfo";
+    @SerializedName(SERIALIZED_NAME_APPLICATION_INFO)
+    private ApplicationInfo applicationInfo;
```

Comment:
Can you double check if ApplicationInfo is correctly populated or if we need to do the same as PaymentsRequest below?
https://github.com/Adyen/adyen-java-api-library/blob/a4b2c054912ca1a6e3a78b6ca1683ab543c0887d/src/main/java/com/adyen/model/checkout/PaymentsRequest.java#L201